documents/devguide/06-pz-logger.md

```diff
@@ -0,0 +1,123 @@
+\# Logger
```

Comment:
The markdown is not actual markdown  because of the slash before the headings (e.g. #, ##).  Verify that the markdown is showing and ping Greg/me to check.

Reply:
Yes… I stand corrected.  I looked at https://github.com/venicegeo/pz-docs/blob/ascii-to-mkdown/documents/userguide/1-introduction.md (when it had the `.markdown` extension) and saw that it didn't seem to be rendered properly; missing the `\`s before the `#`s.